readthedocs/doc_builder/python_environments.py

```diff
@@ -361,7 +361,11 @@ def install_core_requirements(self):
                     negative='sphinx<2',
```

Comment:
I know this isn't inherently related, but I think we should drop the `<2` on the positive here too, and likely consider a newer default.

Reply:
Seems easy enough. I'll make another PR for that.

Reply:
https://github.com/readthedocs/readthedocs.org/pull/7201